rundeckapp/grails-app/migrations/changelog.groovy

```diff
@@ -82,6 +82,13 @@ databaseChangeLog = {
         property name: "varchar3072.type", global: "true", value: "VARCHAR(3072)", dbms: "mysql,postgresql,mssql,h2,mariadb"
         property name: "varchar3072.type", global: "true", value: "VARCHAR2(3072 CHAR)", dbms: "oracle"
 
+        if('${catalogName}' == '${catalogName}'){
```

Comment:
As discussed, i think it isn't worth to support a custom catalog name. I'd just configure the default Catalog name and ditch that overly confusing If clause.